application.go

```diff
@@ -126,8 +126,8 @@ type application struct {
 
 	Annotations_ `yaml:"annotations,omitempty"`
 
-	Constraints_        *constraints                  `yaml:"constraints,omitempty"`
-	StorageConstraints_ map[string]*storageconstraint `yaml:"storage-constraints,omitempty"`
+	Constraints_       *constraints                 `yaml:"constraints,omitempty"`
+	StorageDirectives_ map[string]*storageDirective `yaml:"storage-directives,omitempty"`
```

Comment:
Is this safe to rename now?

Reply:
Yes, the unmarshall handles both the old and the new.